### ISSUE TITLE
AJAX error in Ci Incident edit form.

### ISSUE THREAD

**Issue by solker** [shoov/shoov#37]
While I am trying to fill the input Failing Build or Fixed Build i get an ajax error.

![selection_057](https://cloud.githubusercontent.com/assets/10127295/8022862/cccad800-0cf6-11e5-9304-e89e83e7587e.png)

**Comment by amitaibu** [comment]
Indeed, this is because the Message entity doesn't have a "label". This is a probably an entityreference module bug.
